### PR TITLE
feat: test_fund_manager

### LINKED ISSUE #90: [test] Add all tests for FundManager Contract
### This issue will be part of ODHack8.0, please apply via Onlydust app
- Add a test file for the FundManager contract. It must test all the methods within the contract, including constructors.
- It should follow the standard already set by previous tests in the project.
- The tests should evaluate the success cases of the methods and possible errors in their execution.
- Knowledge in Cairo and experience with scarb and snforge is recommended.
- Please read [contributors guide](https://github.com/web3wagers/gostarkme/blob/dev/CONTRIBUTORS_GUIDE.md) before asking for an issue.

### PR DESCRIPTION
**Pull request by diegoTech14**
# Pull Request

- [ ] Closes #90 
- [x] Added tests (if necessary)
- [x] Run tests
- [ ] Run formatting
- [x] Commented the code

## Changes description

* Create new tests for the FundManager contract
  * New tests to cover all the methods in the contract

## Current output
Tests:
image

## Time spent breakdown

* 1 day
  * The effort involves research about how to implement the tests and taking inspiration issues pr like -#87 to understand a better way to implement the tests.

## Comments

No comments

### CODE REVIEW

File: contracts/tests/test_fund_manager.cairo [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+// *************************************************************************
+//                              TEST
+// *************************************************************************
+
+#[test]
+fn test_constructor() {
+    start_cheat_caller_address_global(OWNER());
+    let (contract_address, fund_class_hash) = __setup__();
+    let fund_manager_contract = IFundManagerDispatcher { contract_address };
```
Comment (EmmanuelAR) [inline contracts/tests/test_fund_manager.cairo:64]: Pending to assert constructor values, like owner and class hash that you set in the constructor, feel free to add get methods to that values to assert.

**Review by EmmanuelAR — CHANGES_REQUESTED** [review]

File: contracts/tests/test_fund_manager.cairo (hunk 1 above)
Comment (diegoTech14) [reply contracts/tests/test_fund_manager.cairo:64]: Got it sir, I'm on it!

Comment (diegoTech14) [reply contracts/tests/test_fund_manager.cairo:64]: I updated the issue with the suggestion and some details were fixed.

**Review by EmmanuelAR — APPROVED** [review]

**Comment by EmmanuelAR** [comment]
LGTM! Excellent work @diegoTech14

**Pull request merged**